website_sale_product_detail_attribute_image/tests/test_website_sale_product_detail_attribute_image.py

```diff
@@ -4,14 +4,89 @@
 
 
 class WebsiteSaleHttpCase(HttpCase):
+    at_install = False
+    post_install=True
+
+    def setUp(self):
+        super().setUp()
+        self.ProductAttribute = self.env['product.attribute']
+        self.ProductAttributeValue = self.env['product.attribute.value']
+        self.ProductAttributeLine = self.env['product.template.attribute.line']
+
+        attribute_dangerous = self.ProductAttribute.create({
```

Comment:
I don't understand... Why are you creating all these records if they're not used in the tests? :thinking: